diff --git a/pyproject.toml b/pyproject.toml
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -70,7 +70,6 @@ dependencies = [
     "mdurl==0.1.2",
     "miniaudio==1.71",
     "mistralai==2.6.0",
-    "mistralai-vibe-local-harness==0.5.1",
     "more-itertools==11.0.2",
     "opentelemetry-api==1.39.1",
     "opentelemetry-exporter-otlp-proto-common==1.39.1",
@@ -81,7 +80,8 @@ dependencies = [
     "packaging==26.2",
     "pexpect==4.9.0",
     "platformdirs==4.9.6",
-    "protobuf==6.33.6",
+    # Needed until https://github.com/conda-forge/protobuf-feedstock/issues/274  is fixed
+    "protobuf",
     "ptyprocess==0.7.0",
     "pyasn1==0.6.4",
     "pyasn1-modules==0.4.2",
